index.html

```diff
@@ -39,6 +42,201 @@ <h4>LET'S CONNECT</h4>
     </div>
     </section>
 
+    <section id="works">
+        <div class="card-work">
+            <img class="work-presentation" src="css/img/Snapshoot1.png">
+            <div class="block">
+                <div class="first-text-section">
+                    <h1>Tonic</h1>
+                    <div class="frame">
+                        <ul>
+                            <li class="frame1">CANOPPY</li>
+                            <li><img class="dot" img src="css/img/Counter.png"></li>
+                            <li class="frame2">Back End Dev</li>
+                            <li><img class="dot" img src="css/img/Counter.png"></li>
+                            <li class="frame3">2015</li>      
+                        </ul>
+                    </div>
+                </div>

```

Comment:
- [ ] Kindly center your cards, like it is in the design

- Figma design
<img width="196" alt="image" src="https://github.com/BiancaMe/Portfolio-Mobile/assets/41163571/4a01a1e5-d1bd-44ce-a36d-fa082948ac7a">

- Your design
<img width="247" alt="image" src="https://github.com/BiancaMe/Portfolio-Mobile/assets/41163571/18451d98-2fe6-4615-8168-e652364fc7ec">